tools/jit/gen_jit_dispatch.py

```diff
@@ -85,16 +85,16 @@ def is_tensor_arg(arg):
         if 'namespace' in decl['method_of']:
             if any(arg['simple_type'] == 'TensorList' for arg in arguments):
                 assert sum(map(is_tensor_arg, arguments)) == 1
-                args = ['as_tensor_list(vars)' if is_tensor_arg(arg) else arg['name']
+                args = ['inputs' if is_tensor_arg(arg) else arg['name']
```

Comment:
No one expects the Spanish Implicit Conversion!!

Reply:
I changed all of these methods to actually work Tensor -> Tensor rather than Variable -> Variable. So this is no longer a cast.